packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -95,7 +95,7 @@ export class ARRenderer extends EventDispatcher {
     await gl.makeXRCompatible();
 
     session.updateRenderState(
-        {baseLayer: new XRWebGLLayer(session, gl, {alpha: true})});
+        {baseLayer: new XRWebGLLayer(session, gl, {alpha: true, framebufferScaleFactor: 0.5})});
```

Comment:
Thanks @klausw I'll definitely be making use of this in the future. Meanwhile, would you mind removing this and pulling master so I can merge the types?

Reply:
Done.